packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cli.integtest.ts

```diff
@@ -1823,6 +1823,41 @@ integTest('hotswap deployment for ecs service detects failed deployment and erro
   expect(deployOutput).not.toContain('hotswapped!');
 }));
 
+integTest('hotswap ECS deployment respects properties override', withDefaultFixture(async (fixture) => {
+  // GIVEN
+  const stackArn = await fixture.cdkDeploy('ecs-hotswap', {
+    captureStderr: false,
+  });
+
+  // WHEN
+  await fixture.cdkDeploy('ecs-hotswap', {
+    options: [
+      '--hotswap',
+      '--hotswap-ecs-minimum-healthy-percent', '100',
+      '--hotswap-ecs-maximum-healthy-percent', '200',
```

Comment:
This should not be a CLI option. The CLI should not understand configuration about AWS resources, because that's what the CDK application is responsible for. Allowing configuration of hotswappable resources like this is not desirable. 

Would adding an entry to `cdk.json` solve this for you? This key should be a boolean; if set, hotswap will use the value configured on the construct, and otherwise it will use the default values. 

Here's an example of adding a key to `cdk.json`: https://github.com/aws/aws-cdk/pull/17176/files.

I could see expanding more hotswap keys here, so `hotswap` should be a top-level key like `watch`, and the min and max healthy percentages should be configurable keys within the `hotswap` entry.

Reply:
Hi @comcalvi ,

We have updated the PR, changing the implementation to get the configuration from cdk.json
Here is an example of the configuration:
`hotswap: {
      ecs: {
        minimumHealthyPercent: ecsMinimumHealthyPercent,
        maximumHealthyPercent: ecsMaximumHealthyPercent,
      },
    },`
    
    Please let me know your thoughts and if we need to change something.
    
    Our change didn't affect tests that create snapshots, but the lint is failing.
    Can you review it please and let me know what the process is.